src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/subheadline/index.js

```diff
@@ -2,10 +2,15 @@ import convertParagraph from '../paragraph';
 import { blockBase } from '#app/models/blocks';
 
 const convertToSubheadline = async block => {
-  const innerParagraph = await convertParagraph(block);
-  return blockBase('subheadline', {
-    blocks: [innerParagraph],
-  });
+  const typesArray = ['crosshead', 'heading', 'subheading'];
+  if (typesArray.includes(block.type)) {
+    const innerParagraph = await convertParagraph(block);
+    return blockBase('subheadline', {
+      blocks: [innerParagraph],
+    });
+  }
+  console.log(`Incorrent block type ${block.type}`);
+  return null;
```

Comment:
I was thinking instead of returning `null`, could we just return the unconverted block back instead here, just a thought

Reply:
We went for this approach to be consistent with how `convertToOptimoBlocks/index.js` handles unsupported block types, it just returns null and presumably the layer above does not add the block to the content:
https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/index.js#L28..L32

For what it's worth I'd be be quite inclined to 
```javascript
throw `Incorrect block type ${block.type}`
```
With the changes, we have made we are narrowing down the component to only work with three types of CPS block, may as well fail out if it is used with any other. I'm not sure if I've seen this type of approach taken in `simorgh` so far though.

Reply:
I think the other block helpers return null if the input is invalid.